plugin/DataView/DataListViewFactory.php

```diff
@@ -73,7 +73,7 @@ public function createListViewByRow(array $row): DataViewFilterableFields
 		$pListView->setAvailableOptions($row['availableOptions']);
 		$pListView->setSortBySetting((int)$row['sortBySetting']);
 		$pListView->setSortByUserDefinedDefault($row['sortByUserDefinedDefault']);
-		$pListView->setSortByUserDefinedDirection($row['sortByUserDefinedDirection']);
+		$pListView->setSortByUserDefinedDirection((int) $row['sortByUserDefinedDirection']);
```

Comment:
Shouldn't this be `(bool)`?

Reply:
This field is enum('0','1') data type and return string when we select it from the database.

Reply:
👍🏻